src/main/java/com/synopsys/integration/jenkins/JenkinsWrapper.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * jenkins-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.jenkins;
+
+import static hudson.model.Items.XSTREAM;
+
+import java.util.Collections;
+import java.util.Optional;
+
+import com.cloudbees.plugins.credentials.Credentials;
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.matchers.IdMatcher;
+
+import hudson.security.ACL;
+import hudson.util.VersionNumber;
+import jenkins.model.Jenkins;
+
+public class JenkinsWrapper {
+
+    private Jenkins jenkins;
+
+    public JenkinsWrapper(Jenkins jenkins) {
+        this.jenkins = jenkins;
+    }
+
+    public static JenkinsWrapper GET_JENKINS_IF_MASTER() {
```

Comment:
This naming might be a little confusing, since it reads as though you are trying to accomplish the same thing as getJenkins just on master. I propose `initializeFromJenkinsJVM()` since JenkinsJVM is an existing utility class in the Jenkins ecosystem that allows the user to check whether they are on master (the Jenkins JVM) or an agent (not the Jenkins JVM -- https://javadoc.jenkins.io/jenkins/util/JenkinsJVM.html